SU-SWS/stanford_magazine
# 7.x 1.x series

# READY FOR PREVIEW
Hey Kellie, since you'll be finishing the work on this, I thought you'd like to test functionality up to this point.
# Summary
- This adds the Magazine Series to Stanford Magazine.

# Needed By (Date)
- Sprint end

# Criticality
- Client is eagerly awaiting this.

# Steps to Test
- Checkout this branch & enable the module
```
cd modules/stanford/stanford_magazine
git pull origin 7.x-1.x-series
git checkout 7.x-1.x-series
drush en stanford_magazine_series -y
drush cc all
```
- Create a new stanford_magazine_series node and select 10 or fewer articles.
- Save your new node
- Verify you can navigate by clicking on the **Jump to next**
- Verify the sticky navigation by scrolling up and down the page
- Verify you can navigate to a selected article. 
-- First get the anchor through inspect. It should look something like `div id="article10650"`
-- then attach it to the end of the URL for your stanford_magazine_series node display. Something like:
`https://sites.stanford.edu/jse-soe-dev/test-series#article10650`

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2374 

## Related PRs

## More Information
There's a demonstration at:
**https://sites.stanford.edu/jse-soe-dev/test-series#article10650**

## Folks to notify
@kbrownell 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
